### PR TITLE
docs: add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by pkalever**
Signed-off-by: Prasanna Kumar Kalever <prasanna.kalever@redhat.com>

### CODE REVIEW

**Comment by pkalever** [comment]
Restoring the conversation here: Closing #194 #196

>> @lxbsz @amarts Please review and help with suggestions.
>>
>> you can have a glance about how it looks like, here:
>> https://github.com/pkalever/gluster-block/blob/contributing-pr/CONTRIBUTING.md
>>
>> Thanks!

> @pkalever
> BTW, should we mention that if adding new features, should the test cases be added at the same time ?
> Thanks.

Good thought, let me add it to the doc. Thanks @lxbsz !

**Comment by pkalever** [comment]
@lxbsz Done with the changes, please review. Thanks!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+* Please follow [submitting new PR guide](https://github.com/gluster/gluster-block/blob/master/CONTRIBUTING.md#did-you-write-a-patch) when your code is ready for review.
+
+#### **Do you have questions about the source code?**
+
+* Ask any question about how to use gluster-block in the [gluster-users mailing list](https://lists.gluster.org/mailman/listinfo/gluster-users).
+
+#### **Do you want to contribute to the gluster documentation?**
+
+* Please read [Contributing to the gluster Documentation](TODO).
```
Comment (pkalever) [inline CONTRIBUTING.md]: @amarts what would be the link replacing TODO above ? https://docs.gluster.org/en/latest/ ? anything specific for block documentation ?

I'm aware @itsravi is working on a quick start guide, not sure where can I locate it.

Thanks!

Comment (amarts) [reply CONTRIBUTING.md]: For now, we can keep this: https://github.com/gluster/glusterfs/tree/master/doc Once Ravi is done with things there, we can always change this link.

Comment (pkalever) [reply CONTRIBUTING.md]: @amarts make-sense!

Will make this changes now. Thanks!

Comment (pkalever) [reply CONTRIBUTING.md]: @amarts updated this, please take a look here:
https://github.com/pkalever/gluster-block/blob/contributing-pr/CONTRIBUTING.md

Comment (pkalever) [reply CONTRIBUTING.md]: All set for merge, just waiting for a waive from @amarts . Thanks!

**Comment by amarts** [comment]
Looks good Prasanna! 

Pressure on us to fix the website (for the 'Join the team here') :-)

**Comment by pkalever** [comment]
Thanks @amarts @lxbsz !

**Pull request merged**